node/pkg/aggregator/node.go

```diff
@@ -15,16 +16,16 @@ import (
 	"github.com/rs/zerolog/log"
 )
 
+const LEADER_TIMEOUT = 5 * time.Second
```

Comment:
@nick-bisonai If I understand it correctly, `LEADER_TIMEOUT` represents the submission frequency to `SubmissionProxy`. Shouldn't we take the value from `orakl-config`?

Reply:
I thought it was unnecessary since it's now batch submitting and don't take different submission frequency per pairs

Reply:
That is true, but maybe we have to come up with some other way how to define the frequency of submissions, because we should be able to support different frequencies eventually.

Reply:
In that case I personally think it could be better to write a new package or service rather than trying to handle every cases from single package. and it's not something which is required at the moment so I'll just keep it in mind. 

As I explained last time, if trying to build something that can handle all the case I think the whole structure or design should be quite different from now and can't be handled by just adjusting single variable into taking from orakl-config